Tasks/AzureVmssDeployment/task.json

```diff
@@ -77,6 +77,21 @@
             }
         },
         {
+            "name": "vmssOsType",
+            "type": "pickList",
+            "label": "OS type",
+            "required": false,
```

Comment:
why is required false?

Reply:
this is for back-compat. currently task does not have os type.

Reply:
changed to required. Muthu said this is fine.